assets/js/pages/ExecutionResults/CheckResultDetail/ExpectedValues.stories.jsx

```diff
@@ -11,8 +10,8 @@ export const Default = {
   args: {
     isTargetHost: true,
     expectedValues: [
-      { name: faker.animal.bear(), value: faker.number.int() },
-      { name: faker.animal.bear(), value: faker.number.int() },
+      executionValueFactory.build({ customized: true }),
```

Comment:
This is an overall question, do we want to use factories in our story? 
In case we introduce changes to the factory, we always need to check the stories where the dedicated factory was used. 
I'm not against it :+1:  , but on the other hand using static values in storybook is more trivial? 
What do you think ?

Reply:
Using factories in stories allows to have stories built with near-real data shape that components use at runtime.

It does not change much either way, imho, because if we change a component in a way that used data is affected we very likely need to change its related story static values or factories.